CMakeLists.txt

```diff
@@ -290,7 +290,15 @@ add_dependency_includes(${OSMIUM_INCLUDE_DIR})
 # Disallow deprecated protozero APIs
 add_definitions(-DPROTOZERO_STRICT_API)
 
+# First get library for just unit_test (this is used in ./unit_tests/CMakeLists.txt)
+set(AllBoostLibrariesExceptUnitTest ${Boost_LIBRARIES})
 
+find_package(Boost 1.49.0 REQUIRED COMPONENTS unit_test_framework)
+
+# After the find_package call we got only the unit test library
+set(BoostUnitTestLibrary ${Boost_LIBRARIES})
+
+# Now get info for the remainder of boost libraries
```

Comment:
CMake's FindBoost module exports

    #   Boost_<C>_LIBRARY      - Libraries to link for component <C>

does this suffice already and spare us to to the find_package-twice-hack?